sphinxcontrib/confluencebuilder/nodes.py

```diff
@@ -36,6 +36,14 @@ class confluence_expand(nodes.Body, nodes.Element):
     using Confluence's expand macro.
     """
 
+class confluence_excerpt(nodes.Body, nodes.Element):
```

Comment:
And in relation to the other comments, maybe `nodes.Element` needs to be changed to `ConfluenceParams`?